### PR TITLE
New component: Floating Action Button (FAB)

### PR DESCRIPTION
**Pull request by splashdust**
The Floating Action Button is used for functions central to the user experience, such as support and help features that needs to be accessible at all times.
 
The button is always floating visible in the users viewport, typically in the lower right corner. This button can be used with an icon only.

### CODE REVIEW

File: apps/core-lib-dev/src/app/datepicker.element.ts [hunk 1]
```diff
@@ -3,8 +3,6 @@ import { customElement, state } from 'lit/decorators.js'
 import { html } from '@sebgroup/green-core/scoping'
 
 import '@sebgroup/green-core/components/datepicker/index.js'
-import * as DatepickerTransitionalStyles from '@sebgroup/green-core/components/datepicker/datepicker.trans.styles.js'
```
Comment (vsjolander) [inline apps/core-lib-dev/src/app/datepicker.element.ts:6]: Was this intentional?

File: apps/core-lib-dev/src/app/calendar.element.ts [hunk 2]
```diff
@@ -3,8 +3,6 @@ import { customElement, state } from 'lit/decorators.js'
 import { html } from '@sebgroup/green-core/scoping'
 
 import '@sebgroup/green-core/components/calendar/index.js'
-import * as CalendarTransitionalStyles from '@sebgroup/green-core/components/calendar/calendar.trans.styles.js'
```
Comment (vsjolander) [inline apps/core-lib-dev/src/app/calendar.element.ts:6]: Was this intentional?

**Review by vsjolander — COMMENTED** [review]
LGTM!

not sure if the removal of transitional styles on other components were intentional?

File: apps/core-lib-dev/src/app/calendar.element.ts (hunk 2 above)
Comment (splashdust) [reply apps/core-lib-dev/src/app/calendar.element.ts:6]: Yes, this is just in `core-lib-dev` in order to test without transitional styles!

**Pull request merged**